mix.exs

```diff
@@ -49,6 +49,7 @@ defmodule Protobuf.Mixfile do
       {:dialyxir, "~> 1.0", only: [:dev, :test], runtime: false},
       {:credo, "~> 1.5", only: [:dev, :test], runtime: false},
       {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
+      {:grpc, "~> 0.6", only: :test},
```

Comment:
I think `:protobuf` can't depend on `:grpc`, because `:grpc` depends on `:protobuf`. This is a somewhat recent development, at `:grpc` 0.6, it did not.

Reply:
Mhmm, this makes it hard to test the grpc codegen (I think, it's been a while since I've looked at this code.) I'll have to think about some other way to make this work nicely.

Reply:
I think an option is to create a separate package to test this. It doesn't look great, but does the job. I'd love to merge this and will try to invest some time to play with your branch over the next weeks (I'm slow, got a lot of work).

Reply:
Noted. I'll try to get to this, though my week is also pretty busy so I can't give a timeline.

Reply:
Hi, @btkostner! 
I just pushed [a branch](https://github.com/elixir-protobuf/protobuf/tree/documentation-2) removing the dependency. I think the test definitely tests less than the one you have here, but I think the parsing functionality is already covered elsewhere, so it's probably fine to do it like I did. WDYT? Should I push these commits into your branch?

Reply:
Code looks good. Push away!